<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectsOfTeacherModule</class>
 <widget class="QWidget" name="SubjectsOfTeacherModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="menuLayout">
     <item>
      <widget class="QFrame" name="additionMenu">
       <layout class="QHBoxLayout" name="additionMenuLayout">
        <item>
         <widget class="QPushButton" name="confirmAddition">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="abortAddition">
          <property name="text">
           <string>Abort</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addSubject">
       <property name="text">
        <string>Add subject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSubject">
       <property name="text">
        <string>Delete subject</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="subjectsList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
